projects/common/src/navigation/navigation.config.ts

```diff
@@ -1,13 +1,13 @@
-import { Color, FeatureState } from '@hypertrace/common';
 import { Observable } from 'rxjs';
-import { IconSize } from '../icon/icon-size';
+import { Color } from '../color/color';
+import { FeatureState} from '../feature/state/feature.state';
 
 export type NavItemConfig = NavItemLinkConfig | NavItemHeaderConfig | NavItemDividerConfig;
 
 export interface NavItemLinkConfig {
   type: NavItemType.Link;
   icon: string;
-  iconSize?: IconSize;
+  iconSize?: string;
```

Comment:
I don't think we should remove this type safety. We only support few size options. Moving the method to a new service would solve this.